webrender/src/border.rs

```diff
@@ -544,3 +674,43 @@ impl BorderCornerDashClipData {
         }
     }
 }
+
+/// Represents the GPU data for drawing a single dot
+/// to a clip mask.
+#[derive(Debug, Clone)]
+#[repr(C)]
+pub struct BorderCornerDotClipData {
+    pub center: LayerPoint,
+    pub radius: f32,
+    pub padding: [f32; 5],
```

Comment:
It's placed in the same store as the Dash data struct (and the other clip-related GPU structs), so the size has to match.